content/en/observability_pipelines/monitoring.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Monitoring
+disable_toc: false
+further_reading:
+- link: "observability_pipelines/set_up_pipelines"
+  tag: "Documentation"
+  text: "Set up a pipeline"
+- link: "/monitors/types/metric/"
+  tag: "Documentation"
+  text: "Configuring a metric monitor"
+---
+
+## Overview
+
+A pipeline consists of components that collect, process, and route your observability data. You can track the status of your pipelines and components by:
+
+- Viewing health graphs of your [pipelines](#view-the-status-of-your-pipelines), [Workers](#view-the-status-of-your-workers), and [components](#view-the-status-of-your-pipeline-components) (sources, processors, and destinations).
+- Enabling [out-of-the-box monitors](#out-of-the-box-monitors) that alerts you if:
+    - An Observability Pipelines Worker has high CPU usage, high memory usage, or is dropping data.
+    - A component is emitting errors.
+    - A defined quota has been reached.
+- Creating your own dashboards, notebooks, and monitors with the available [metrics](#metrics).
+
+## View the status of your pipelines
+
+1. Navigate to [Observability Pipelines][1] to see how many bytes your pipelines are receiving and sending out.
+1. Select your pipeline.
+1. Click the **Health** tab to see graphs of:
```

Comment:
Not sure if this is actually available for everyone, or if it is behind a feature flag.

Reply:
It's available now.